src/StaticAssets/src/Infrastructure/StaticAssetsEndpointDataSourceHelper.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.StaticAssets.Infrastructure;
+
+/// <summary>
+/// This type is not recommended for use outside of ASP.NET Core.
+/// </summary>
+public static class StaticAssetsEndpointDataSourceHelper
+{
+    /// <summary>
+    /// This method is not recommended for use outside of ASP.NET Core.
+    /// </summary>
+    /// <param name="dataSource"></param>
+    /// <param name="staticAssetsManifestPath"></param>
+
```

Comment:
```suggestion
    /// <summary>
    /// For internal framework use only.
    /// </summary>
```
Only because a customer might otherwise read this and think they're in the clear because "well _I'm_ using ASP.NET Core..."